src/graph/groups/groups.interface.ts

```diff
@@ -26,7 +26,15 @@ export interface IGroupsAPI {
   getGroupLocationLoader(groupId: number): Promise<any>;
   getGroupAttributesLoader(id: number): Promise<any>;
   createGroup(groupInput: IGroupInput): any;
-  addParticipantToGroup(participantId: number, groupId: number, roleId: number): Promise<GroupParticipant>
+  addParticipantToGroup(
+    participantId: number,
+    groupId: number,
+    roleId: number
+  ): Promise<GroupParticipant>;
+  setGroupParticipantEndDate(
+    groupId: number,
+    id: number
+  ): Promise<GroupParticipant>;
```

Comment:
See comment on groups.api.ts. Can this be 
```suggestion
  setGroupParticipantEndDate(
    groupId: number,
    participantId: number
  ): Promise<GroupParticipant>;
```
?